frontend/to-do-frontend/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
nit: puede ser en el siguiente pr, puedes referenciar este en el readme que esta en la raiz, moverlo o modificarlo de acuerdo a tu proyecto.

una lectura opcional: https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/